Examples/Python/python/acts/examples/simulation.py

```diff
@@ -616,7 +616,7 @@ def getG4DetectorConstructionFactory(
 def addGeant4(
     s: acts.examples.Sequencer,
     detector: Optional[Any],
-    trackingGeometry: Union[acts.TrackingGeometry, acts.Detector],
+    recoGeometry: Union[acts.TrackingGeometry, acts.Detector],
```

Comment:
Personally I would stick with `trackingGeometry` even with the Gen2 because it feels more descriptive to me and includes the "reconstruction" context

Reply:
ok, sounds good !